content/en/tracing/guide/setting_primary_tags_to_scope.md

```diff
@@ -80,6 +80,8 @@ Go to the [APM Settings][6] page to define, change, or remove your primary tags.
 * Changes may take up to two hours to be reflected in the UI.
 * The tracer always adds `resource`, `name`, and `service` tags to spans. Datadog recommends never adding these as host level tags to avoid confusion.
 * The second primary tag supports up to 30 unique values. See [APM Data Volume Guidelines][9] for details.
+* Adding second primary tags is currently not supported for Serverless environments.
```

Comment:
What do you think about making this a callout note at the top of the section? It's not really parallel to the other information in these bullet points, and it makes sense to disclose it before the user can even try to go set a second primary tag.